src/Cake.Web.Core/Content/Modules/Module.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using Cake.Web.Core.NuGet;
+using Cake.Web.Docs;
+
+namespace Cake.Web.Core.Content.Modules
+{
+    public sealed class Module : IDocumentationMetadata
+    {
+        public string Name { get; set; }
+        public Uri Website { get; set; }
+        public Uri Repository { get; set; }
+        public string Author { get; set; }
+        public string Description { get; set; }
+        public List<string> Categories { get; set; }
+        public PackageDefinition PackageDefinition { get; set; }
+
+        public Uri Url => Repository ?? Website;
+        public bool IsOwnedByAddin => true;
```

Comment:
We should add an IsOwnedByModule and so we i.e. can add Modules at
https://github.com/cake-build/website/blob/97fb9986af1f233f87caeb8def3bfeaad4c58225/src/Cake.Web/Views/Dsl/Category.cshtml#L35